.circleci/config.yml

```diff
@@ -47,7 +47,7 @@ jobs:
     - run:
         name: Install bundle
         command: gem update --system && gem install bundler && bundle update
-    - run: bundle exec rake circleci:post
+    # - run: bundle exec rake circleci:post
```

Comment:
Did you mean to revert this?

Reply:
Yes. I don't want both kokoro and circle to build and push docs.